packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -114,16 +120,24 @@ export const Summary = styled.p`
   }
 `;
 
-const StoryPromo = ({ image, info }) => (
+const StoryPromo = ({ image, info, mediaIndicator }) => (
   <StoryPromoWrapper>
-    <ImageGridItem>{image}</ImageGridItem>
+    <ImageGridItem>
+      {image}
+      <InlineMediaIndicator>{mediaIndicator || null}</InlineMediaIndicator>
```

Comment:
Shouldn't we print the `InlineMediaIndicator` just if the `mediaIndicator` prop is passed?

```suggestion
    {mediaIndicator && (
        <InlineMediaIndicator>{mediaIndicator}</InlineMediaIndicator>
      )}
```